FirebaseAppCheck/Interop/FIRAppCheckInterop.h

```diff
@@ -31,6 +31,10 @@ NS_SWIFT_NAME(AppCheckInterop) @protocol FIRAppCheckInterop
                     completion:(FIRAppCheckTokenHandlerInterop)handler
     NS_SWIFT_NAME(getToken(forcingRefresh:completion:));
 
+/// Retrieve a new limitedUseToken
+- (void)getLimitedUseTokenWithCompletion:(FIRAppCheckTokenHandlerInterop)handler
+    NS_SWIFT_NAME(getlimitedUseToken(completion:));
```

Comment:
nit: capitalization
```suggestion
    NS_SWIFT_NAME(getLimitedUseToken(completion:));
```

Reply:
Ah thank you and sry for the style stuff